static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -0,0 +1,32 @@
+# Caveats running DVC on Windows
+
+There can be few performance issues while running DVC on Windows. Below are some
+of these issues detected so far. If you find any, please let us know!
+
+## Caveat 1: DVC binaries are slow on Windows
```

Comment:
No need to prefix every title with `Caveat #`. It's clear from the context.

Reply:
it's not about binaries. DVC is slow in general on Windows in general:

1. NTFS issue. We need a link to StackOverflow or some other resource.
2. Windows defender is enabled by default. It makes DVC extremely slow. It should be whitelisted. We need to include a link or an instruction.
3. 8dot3 (might be related to the 1)

Reply:
Done